### PR TITLE
build(Dependencies): update iterable android and iOS dependencies

### PR DESCRIPTION
**Pull request by danielgomezrico**
The project have iOS and Android dependencies outdated:

- iOS Changelog: https://github.com/Iterable/swift-sdk/releases
- Android Changelog: https://github.com/Iterable/iterable-android-sdk/releases

### CODE REVIEW

**Comment by danielgomezrico** [comment]
I tested on each platform that pushes are still working after this

File: android/build.gradle [hunk 1]
```diff
@@ -42,9 +42,7 @@ android {
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
 
-    implementation 'com.iterable:iterableapi:3.4.2'
-    implementation 'com.google.firebase:firebase-messaging:22.0.0'
```
Comment (cdmunoz) [inline android/build.gradle:46]: Is this implementation not required anymore?

**Review by cdmunoz — APPROVED** [review]
LGTM 👌🏻

I left you a small question

**Review by juansoos — APPROVED** [review]
OSOM 💯

File: android/build.gradle (hunk 1 above)
Comment (danielgomezrico) [reply android/build.gradle:46]: I tested and it works fine without that, why was that required?

**Pull request merged**